docs/source/getting_started/04_benchmarks.rst

```diff
@@ -6,75 +6,8 @@ At PeekingDuck, we aim to give users as much flexibility as possible to cater to
 From real-time object detection to high-accuracy pose estimation, PeekingDuck houses a wide range of models.
 The full list of models can be found :mod:`here <peekingduck.pipeline.nodes.model>`.
 
-
-In this document, we provide a summary of the model performance and FPS based on our own tests, and careful curation of the test conditions.
-The purpose of these benchmarks is to allow users to have a quick understanding on the relative performance of each model, thereby enabling
+We are currently running benchmarks on model performance and FPS, and will be updating this document with our benchmark results.
+We hope that this would allow users to have a quick understanding on the relative performance of each model, thereby enabling
 users to decide on the model most suited for their use case.
 
-However, this document is **NOT** a guaratee of performance for your own use case, as there are many variables that may affect performance.
-
```

Comment:
I suggest we maintain this page as it is, then create a `WIP.md` and link the benchmarks to this WIP MD. Adding tables to rst is quite tedious.